.eslintrc.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
As you fix the linting issues, could you please update this to our latest linting style using the project-wide shared configuration? Example is at https://github.com/fluid-project/kettle/blob/master/.eslintrc.json